aiozk/client.py

```diff
@@ -119,7 +112,8 @@ async def close(self):
     def wait_for_events(self, event_types, path):
         path = self.normalize_path(path)
 
-        f = self.loop.create_future()
+        loop = asyncio.get_running_loop()
```

Comment:
Perhaps there was miscommunication about `asyncio.get_running_loop`. I should be more clear.
I didn't ask you to replace all `asyncio.get_event_loop` other than I mentioned explicitly. What I meant was that `asyncio.get_ruuning_loop` is preferred than `asyncio.get_event_loop` if it is called directly inside coroutines or callbacks. Normal functions can be called outside async context, in this case `asyncio.get_running_loop` fails.

But I expect all replaced `asyncio.get_running_loop` would be working too, because I anticipate that aiozk users only call coroutine functions offered by ZKClient or at least they use aiozk inside async context. It guarantees that all callee code of aiozk will run in async context so `asyncio.get_running_loop` should not fail.

I don't want to bother you for reverting this. And I think that doing so seems too pedantic and nothing valuable can be gained. :smile: